presto-orc/src/main/java/com/facebook/presto/orc/metadata/ColumnStatistics.java

```diff
@@ -32,13 +37,51 @@ public ColumnStatistics(
             DateStatistics dateStatistics,
             DecimalStatistics decimalStatistics)
     {
+        this(numberOfValues, booleanStatistics, integerStatistics, doubleStatistics, stringStatistics, dateStatistics, decimalStatistics, true, null);
```

Comment:
I was in the assumption that it would create garbage although looking at the source it's a static final https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/RegularImmutableList.java#L30